pkg/apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -48,6 +48,13 @@ const (
 	PodIdentityProviderAzure                      = "azure"
 	PodIdentityProviderGCP                        = "gcp"
 	PodIdentityProviderSpiffe                     = "spiffe"
+	PodIdentityProviderEKS                        = "eks"
```

Comment:
Thinking out loud - Since it's AWS official approach, maybe rename this to AWS and not specifically AWS EKS?

For example, what if they support it else where?

Reply:
I went back and forth on that a couple times. We could use 'aws-eks' and 'aws-kiam' since both are aws specific.

Reply:
I like that approach!